index.html

```diff
@@ -157,6 +157,30 @@
             <label for="filled-checkbox">Filled fields?</label>
             <i class="fa fa-question-circle has-tooltip" title="shows sample text in all [field]s"></i>
         </div>
+        <div class="input-field">
+            <label for="pen-style-select">Pen style</label>
+            <select id="pen-style-select">
+                <option value="Verdana, sans-serif" selected>Pen</option>
+                <option value="Comic Sans MS, sans-serif">Crayon</option>
+                <option value="Segoe Script, sans-serif">Fancy</option>
```

Comment:
The fancy pen can only be #1c1713 and no other pens have that color (except a rainbow crayon set to exactly that hex, I suppose.)

If we don't want to handle populating that select just yet (and populating selectmenus in jQuery UI involves some frustrating issues, not certain glancing at this whether or not it ends up in a selectmenu by being in this dialog) then perhaps when 'fancy' is selected, the other select should automatically default to selecting the 'fancy' color.